docs/advanced-usage.md

```diff
@@ -86,7 +86,7 @@ steps:
     node-version: '14'
     cache: 'npm'
     cache-dependency-path: '**/package-lock.json'
-- run: npm install
+- run: npm ci
```

Comment:
Does it make sense to use `npm ci` when the dependencies are being restored from a cache?

Reply:
I believe the cache option only caches the global cache, not the `node_module` directory. So it still makes sense.

Reply:
It does look like it's running `npm config get cache` to get the path to cache, so that does make sense, thanks 👍